src/components/vue-pagination-tw.vue

```diff
@@ -162,8 +162,8 @@ const calculatedPages = computed(() => {
           aria-current="page"
           class="bg-white border border-gray-300 text-gray-500 relative inline-flex items-center px-4 py-2 text-sm font-medium"
           v-if="hasLast()"
-          @click.prevent="changePage(totalPages)"
-        >{{ totalPages }}</a>
+          @click.prevent="changePage(totalPages.value)"
+        >{{ totalPages.value }}</a>
```

Comment:
```suggestion
          @click.prevent="changePage(totalPages)"
        >{{ totalPages }}</a>
```

Reply:
Nicely work!

I've tried to build this PR and found the error that `.value` was in the template syntax. I removed them then everything worked as expected.